src/client/players.py

```diff
@@ -102,6 +102,11 @@ def get(self, item, default):
         val = self.__getitem__(item)
         return val if val else default
 
+    def getID(self, name):
+        if name in self._logins:
+            return self._logins[name].id
+        return -1
```

Comment:
This should definitely log something if it can't find an ID. The current behaviour will cause an obscure crash in whatever called getID when the -1 is eventually used, without leaving a nice easy to follow trail.

I suggest logging something at the WARN level. Or maybe even throw an exception?